scripts/standin_events_extractor.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+
+"""
+python standin_events_extractor.py \
+--start-date 2025-03-01 \
+--end-date 2025-03-31 \
+--cosmos-connection-string "your-conn-str"
+"""
+
+import argparse
+import csv
+import datetime
+import logging
+import sys
+from datetime import timezone
+
+from azure.cosmos import CosmosClient
+
+# Configurazione del logging
+logging.basicConfig(
+    level=logging.ERROR,
+    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+)
+logger = logging.getLogger("standin-events-extractor")
+logger.setLevel(logging.INFO)
+
+
+def parse_arguments():
+    """Parse command line arguments"""
+    parser = argparse.ArgumentParser(description='Extract stand-in events from Cosmos DB')
+
+    # Parametri per date e output
+    parser.add_argument('--start-date', required=True, help='Start date in YYYY-MM-DD format')
+    parser.add_argument('--end-date', required=True, help='End date in YYYY-MM-DD format')
+    parser.add_argument('--output', default='standin_events.csv', help='Output CSV file name')
+
+    # Parametri per connessione a Cosmos DB
+    parser.add_argument('--cosmos-connection-string', required=True,
+                        help='Cosmos DB connection string')
+    parser.add_argument('--cosmos-database', default='standin',
+                        help='Cosmos DB database name')
+    parser.add_argument('--cosmos-container', default='events',
+                        help='Cosmos DB container name')
+
+    return parser.parse_args()
+
+
+def get_cosmos_client(args):
+    """Create and return a Cosmos DB client using connection string"""
+    try:
+        connection_string = args.cosmos_connection_string
+        client = CosmosClient.from_connection_string(connection_string)
```

Comment:
to avoid abuse RU in prod env , use `preferred_locations` to switch report query on read replica
 https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/best-practice-python#sdk-usage

[example see here ](https://github.com/pagopa/pagopa-receipt-pdf-generator/commit/872be01c62ca75cc2b0c97ce26e7012e92a06a5e#diff-5ac37c1be518e1a1bb874ba02b3dcaa63e4848f5f95e07eb8fa1f86fb3f8ad89R25-R32)

Reply:
thank you for this advice